apps/site/assets/ts/stop/components/DeparturesAndMap.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement, useState } from "react";
+import { Polyline } from "leaflet";
+import { chain } from "lodash";
+import { DirectionId, Route, Stop } from "../../__v3api";
+import { ScheduleWithTimestamp } from "../../models/schedules";
+import StopPageDepartures from "./StopPageDepartures";
+import StopMapRedesign from "./StopMapRedesign";
+import { useRoutesByStop } from "../../hooks/useRoute";
+
+interface DeparturesAndMapProps {
+  routes: Route[];
+  stop: Stop;
+  schedules: ScheduleWithTimestamp[];
+  lines: Polyline[];
+}
+
+const DeparturesAndMap = ({
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  routes,
+  stop,
+  schedules,
+  lines
+}: /* eslint-enable @typescript-eslint/no-unused-vars */
+DeparturesAndMapProps): ReactElement<HTMLElement> => {
+  const [state, setState] = useState<{
+    email: null | string;
+    password: null | string;
+  }>({ email: null, password: null });
+
+  const [departureInfo, setDepartureInfo] = useState<{
+    departureRoute: Route | null;
+    departureDirectionId: DirectionId | null;
+    departureSchedules: ScheduleWithTimestamp[] | null;
```

Comment:
This should be an array of `DepartureInfo`s as that will hold both the prediction and schedule information.